dist/topology-graph.css

```diff
@@ -21,22 +21,22 @@ kubernetes-topology-graph {
 
```

Comment:
Strangely enough, bower expects us to have changes in the dist directory committed. But yes, they should be generated with ```grunt build```.